File: src/main/java/br/com/robertodebarba/jersey_example_project/api/BaseApi.java
package br.com.robertodebarba.jersey_example_project.api;

import javax.ws.rs.GET;
import javax.ws.rs.Path;
import javax.ws.rs.Produces;
import javax.ws.rs.core.MediaType;

@Path("/")
@Produces(MediaType.TEXT_PLAIN)
public class BaseApi {

	@GET
	public String helloWorld() {
		return "Hello World";
	}

}

File: backend/distributed-flood-monitoring-system-api/src/main/java/br/com/robertodebarba/floodmonitoring/api/BaseApi.kt
package br.com.robertodebarba.floodmonitoring.api

import javax.ws.rs.GET
import javax.ws.rs.Path
import javax.ws.rs.Produces
import javax.ws.rs.core.MediaType

@Path("/")
@Produces(MediaType.TEXT_PLAIN)
class BaseApi {

    @GET
    fun helloWorld(): String {
        return "Hello World"
    }

}
